docs/manual-CN/5.appendix/gremlin-ngql.md

```diff
@@ -0,0 +1,247 @@
+# Gremlin 和 nGQL 对比
+
+## Gremlin 介绍
+
+[Gremlin](https://tinkerpop.apache.org/gremlin.html) 是 Apache ThinkerPop 框架下的图遍历语言。Gremlin 可以是声明性的也可以是命令性的。虽然 Gremlin 是基于 Groovy 的，但具有许多语言变体，允许开发人员以 Java、JavaScript、Python、Scala、Clojure 和 Groovy 等许多现代编程语言原生编写 Gremlin 查询。
+
+## nGQL 介绍
+
+**Nebula Graph** 的查询语言为 [nGQL](../1.overview/1.concepts/2.nGQL-overview.md)， 是一种类 SQL 的声明型的文本查询语言。相比 SQL，`nGQL` 具有如下特点：
+
+- 类 SQL，易学易用
+- 可扩展
+- 大小写不敏感
+- 支持图遍历
+- 支持模式匹配
+- 支持聚合运算
+- 支持图计算
+- 支持分布式事务（开发中）
+- 无嵌入支持组合语句，易于阅读
+
+## 基本概念对比
+
+名称               | Gremlin | nGQL           |
+-----              |---------|   -----       |
+vertex, node       | vertex  | vertex        |
+edge, relationship | edge    | edge          |
+vertex type        | label   | tag           |
```

Comment:
In ngql, the vertex type is tag 

Anyway, I prefer vertex type because a tag really gets me confusing at the very beginning

Reply:
Yes, I agree with you.